### DISCUSSION TITLE
If I mark a movie as watched in Plex, is it added to the viewing history in Tract.tv?

### DISCUSSION THREAD

**Discussion by bulletproof2k** [Taxel/PlexTraktSync#1137] [Support] [answered]
I just marked the movie in plex as viewed, but it did not appear in the tract viewing history.

**Comment by simonc56** [comment]
Steps to reproduce please

**Reply by bulletproof2k** [reply]
As there was no such viewing. I added the movie to the watchlist and then marked it as watched. Should it appear in the watching history in the trakt after running the script?

**Reply by simonc56** [reply]
It's a movie you don't have on your Plex server. Only in your Plex watchlist ?

**Reply by bulletproof2k** [reply]
Yep

**Reply by simonc56** [reply]
Then it's normal. Plex watchlist is not read during watched status sync. Only your libraries are read.

I suggest you open a [Feature request](https://github.com/Taxel/PlexTraktSync/issues/new?assignees=&labels=enhancement&template=feature.md&title=Feature+request%3A) about it.

**Reply by bulletproof2k** [reply]
Ok, thx.

**Reply by cjshrader** [reply]
Was this feature implemented? Or was the feature request even made? This is the exact functionality I'm looking for

**Reply by simonc56** [reply, marked as answer]
work in progress : https://github.com/Taxel/PlexTraktSync/pull/1676

**Reply by cjshrader** [reply]
Thank you very much!

**Comment by glensc** [comment]
closing in favor of feature request
